libs/form-builder/tsconfig.json

```diff
@@ -3,7 +3,9 @@
   "files": [],
   "include": [],
   "compilerOptions": {
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
+    "strict": true,
+    "strictPropertyInitialization": false
```

Comment:
Enabled more strict settings because postybirb-ui with these settings depends on form-builder and thus typecheck produced type errors